src/Parser/DemonologyWarlock/Modules/Items/Legendaries/SindoreiSpite.js

```diff
@@ -0,0 +1,46 @@
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+import { formatNumber } from 'common/format';
+
+import getDamageBonus from '../../WarlockCore/getDamageBonus';
+
+const SINDOREI_SPITE_DAMAGE_BONUS = 0.15;
+
+class SindoreiSpite extends Module {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  bonusDmg = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasWrists(ITEMS.SINDOREI_SPITE.id);
+  }
+
+  on_byPlayerPet_damage(event) {
+    if (this.combatants.selected.hasBuff(SPELLS.SINDOREI_SPITE_BUFF.id, event.timestamp)) {
+      this.bonusDmg += getDamageBonus(event, SINDOREI_SPITE_DAMAGE_BONUS);
+    }
+  }
+
+  on_byPlayer_damage(event) {
+    if (!this.combatants.selected.hasBuff(SPELLS.SINDOREI_SPITE_BUFF.id, event.timestamp)) {
+      return;
+    }
+    // so friendly TKC damage doesn't get counted
+    if (!event.targetIsFriendly) {
+      this.bonusDmg += getDamageBonus(event, SINDOREI_SPITE_DAMAGE_BONUS);
+    }
+  }
+
+  item() {
+    return {
+      item: ITEMS.SINDOREI_SPITE,
+      result: `${formatNumber(this.bonusDmg)} damage contributed - ${this.owner.formatItemDamageDone(this.bonusDmg)}`,
```

Comment:
Use formatItemDamageDone() from the core CombatLogParser instead

Reply:
I'd like to show the damage contributed anyway, perhaps move it to tooltip? Same with Wakener's Loyalty.

Reply:
Oh wow for some reason I didn't even see that you had format item damage done as like the second format. I'm spacing out today. Yeah I think formatItemDamageDone should be solo here, and other notes should be in the tooltip.

Reply:
Ok, I'll rewrite it here, and in second PR I'm working on right now, I'll refactor it in other item modules as well (there will be other refactoring as well so I'll do that as well).

Reply:
Thanks! I'm determined to see a common format for showing Legendary/Set damage contribution. If you really think raw damage should show too, maybe make the argument for a change to formatItemDamageDone.

Reply:
I wouldn't mind being in the format as well, but I think it can stay in tooltip and show percentage / DPS by default.